libs/backend-ddd/src/opportunity/infrastructure/api/brevo/types.ts

```diff
@@ -67,6 +67,12 @@ export interface DealAttributes {
 
 export interface DealUpdateAttributes {
   envoy: boolean
+  deal_stage?: string
```

Comment:
suggestion: `dealStage` semble pouvoir être un valeur de l'enum ?
```suggestion
  deal_stage?: BrevoStatus
```